Drupal\Core\Entity\Exception\NoCorrespondingEntityClassException: The Drupal\user\Entity\User class does not correspond to an entity type. in                                              [error]
Drupal\Core\Entity\EntityTypeRepository->getEntityTypeFromClass() (line 114 of /data/disk/o5372971148/static/9d_commerce_v1.57-prod/core/lib/Drupal/Core/Entity/EntityTypeRepository.php)
#0 /data/disk/o5372971148/static/9d_commerce_v1.57-prod/core/lib/Drupal/Core/Entity/EntityBase.php(487): Drupal\Core\Entity\EntityTypeRepository->getEntityTypeFromClass()
#1 /opt/tools/drush/8/drush/lib/Drush/User/User8.php(26): Drupal\Core\Entity\EntityBase::load()
#2 /opt/tools/drush/8/drush/lib/Drush/Boot/DrupalBoot.php(558): Drush\User\User8->load_by_uid()
#3 /opt/tools/drush/8/drush/includes/bootstrap.inc(354): Drush\Boot\DrupalBoot->bootstrap_drupal_login()
#4 /opt/tools/drush/8/drush/includes/bootstrap.inc(509): drush_bootstrap()
#5 /opt/tools/drush/8/drush/includes/bootstrap.inc(456): drush_bootstrap_max()
#6 /opt/tools/drush/8/drush/lib/Drush/Boot/BaseBoot.php(59): drush_bootstrap_to_phase()
#7 /opt/tools/drush/8/drush/includes/preflight.inc(67): Drush\Boot\BaseBoot->bootstrap_and_dispatch()
#8 /opt/tools/drush/8/drush/drush.php(12): drush_main()
#9 {main}.
Drupal\Core\Entity\Exception\NoCorrespondingEntityClassException: The Drupal\user\Entity\User class does not correspond to an entity type. in Drupal\Core\Entity\EntityTypeRepository->getEntityTypeFromClass() (line 114 of /data/disk/o5372971148/static/9d_commerce_v1.57-prod/core/lib/Drupal/Core/Entity/EntityTypeRepository.php).
Drush command terminated abnormally due to an unrecoverable error.                                                                                                                         [error]    
Drush was not able to start (bootstrap) Drupal.                                                                                                                                            [error]
Hint: This error can only occur once the database connection has already been successfully initiated, therefore this error generally points to a site configuration issue, and not a
problem connecting to the database.

Drush was attempting to connect to:
 Drupal version           :  9.4.8
 Site URI                 :  http://raynoautomotive.com
 Database driver          :  mysql
 Database hostname        :  localhost
 Database port            :  3306
 Database username        :  raynoautomotivec
 Database name            :  raynoautomotivec
 Database                 :  Connected
 Default theme            :  bootstrap_rayno
 Administration theme     :  adminimal_wwm
 PHP executable           :  /opt/php74/bin/php
 PHP configuration        :  /home/o5372971148.ftp/.drush/php.ini
 PHP OS                   :  Linux
 Drush script             :  /opt/tools/drush/8/drush/drush.php
 Drush version            :  8.4.10
 Drush temp directory     :  /home/o5372971148.ftp/.tmp
 Drush configuration      :  /data/disk/o5372971148/static/9d_commerce_v1.57-prod/sites/raynoautomotive.com/drushrc.php
                             /data/disk/o5372971148/static/9d_commerce_v1.57-prod/sites/all/drush/drushrc.php
 Drush alias files        :  /home/o5372971148.ftp/.drush/2022-10-20.a.4.dev.vfpchapter10.org.alias.drushrc.php "
                             There are 47 more alias files. Run with --full to see the full list."
 Install profile          :  standard
 Drupal root              :  /data/disk/o5372971148/static/9d_commerce_v1.57-prod
 Drupal Settings File     :  sites/raynoautomotive.com/settings.php
 Site path                :  sites/raynoautomotive.com
 File directory path      :  sites/raynoautomotive.com/files
 Private file directory   :  sites/raynoautomotive.com/private/files
 path
 Sync config path         :  sites/raynoautomotive.com/private/config/sync